test/system/012-manifest.bats

```diff
@@ -243,6 +243,18 @@ function manifestListAddArtifactOnce() {
     run_podman rmi $list localhost:${PODMAN_LOGIN_REGISTRY_PORT}/test
 }
 
+@test "manifest list inspect matches skopeo" {
+    image=quay.io/giuseppe/zstd-chunked:fedora-manifest
+    run skopeo inspect --tls-verify=false --raw docker://${image}
+    assert $status -eq 0
+    index="$(jq -S <<<$output)"
+    run_podman pull -q ${image}
+    run_podman manifest inspect ${image}
+    output="$(jq -S <<<$output)"
+    assert "$index" == "$output"  "skopeo and podman manifest inspect should match"
```

Comment:
I am not sure this is a realistic requirement and will certainly cause issues if either only podman or skopeo changes, this means it makes it very hard to update if the output ever changes as podman and skopeo are never updated at the same time in our CI or gating.

IMO there is no reason to use skopeo at all, we should check the annotations are listed because otherwise if they are removed from the image the test will be a NOP without us noticing.

Reply:
`skopeo inspect --raw` will reproduce the original on-registry blob exactly.

The Podman code here does _not_ do that, in general (it might differ in white space or if the manifest adds new JSON fields not known to Podman’s data types). It might work well enough in practice, though I _would_ be happier with a JSON-aware comparison instead of a raw byte equality.

Reply:
Added JSON specifics.